modules/RTC/JitsiLocalTrack.js

```diff
@@ -777,6 +871,14 @@ export default class JitsiLocalTrack extends JitsiTrack {
         // we aren't receiving any data from the source. We want to notify
         // the users for error if the stream is muted or ended on it's
         // creation.
+
+        // For video blur enabled use the original video stream
+        if (this._videoBlurEnabled === true) {
```

Comment:
How about the following implementation:
```
   const stream =  this._videoBlurEnabled ? this._originalVideoStream : this.stream;

        return stream.getTracks().some(track =>
            (!('readyState' in track) || track.readyState === 'live')
                && (!('muted' in track) || track.muted !== true));
```

Reply:
Also in this case I guess you have to check if this._originalVideoStream exists...

Reply:
Renamed _originalVideoStream to _originalStream for general use so that no check is needed